org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/semantictokens/PositionReader.java

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
The new files would need the EPL 2.0 header

Reply:
done